eva-accession-pipeline/src/main/java/uk/ac/ebi/eva/accession/pipeline/io/SummaryFileWriter.java

```diff
@@ -0,0 +1,9 @@
+package uk.ac.ebi.eva.accession.pipeline.io;
+
+import org.springframework.batch.item.file.FlatFileItemWriter;
+
+import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
+
+public class SummaryFileWriter extends FlatFileItemWriter<ISubmittedVariant>{
```

Comment:
this will actually be a composite writer. the FlatFileItemWriter will be one of the inner writers

Reply:
agreed to implement `ItemWriter<VariantEntity>`